src/components/Hero/index.tsx

```diff
@@ -2,27 +2,25 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { LocalizedLink as Link } from 'gatsby-theme-i18n';
 import { useDetectOs } from '../../hooks/useDetectOs';
-import type { NodeReleaseLTSVersion } from '../../types';
+import type { NodeReleaseData } from '../../types';
 import styles from './index.module.scss';
 
 interface Props {
   title: string;
   subTitle?: string;
-  nodeReleasesLTSVersion: NodeReleaseLTSVersion[];
+  nodeReleaseData: NodeReleaseData[];
 }
 
-const Hero = ({
-  title,
-  subTitle,
-  nodeReleasesLTSVersion,
-}: Props): JSX.Element => {
+const Hero = ({ title, subTitle, nodeReleaseData }: Props): JSX.Element => {
   const { getDownloadLink } = useDetectOs();
 
-  const [currentRelease, ...releases] = nodeReleasesLTSVersion;
-  const currentLTS = releases.find(release => !!release.lts) || currentRelease;
+  const currentLTS = nodeReleaseData.find(release => release.isLts);
+  const currentRelease = nodeReleaseData.find(
+    release => release.status === 'Current'
+  );
 
-  const ltsVersionUrl = getDownloadLink(currentLTS?.version || '');
-  const currentVersionUrl = getDownloadLink(currentRelease?.version || '');
+  const ltsVersionUrl = getDownloadLink(currentLTS?.release || '');
```

Comment:
I don't understand why you are changing this naming. The variable is ltsVersionUrl but you are calling it `.release` The value is the version of node.js 🤷🏻‍♂️

Reply:
Because of the data that we get with @pkgjs/nv, it was easier to have `release` as the full release version and `version` as the short version. 

It is of course a matter of naming, but replacing the otherwise would give way more trouble 😅  (I could rename the `release` into `fullVersion` or something.